config/pipeline.yaml

```diff
@@ -920,6 +920,26 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.cpufreq
 
+  kselftest-cpufreq-suspend:
+    template: kselftest.jinja2
+    kind: job
+    params:
+      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
+      collections: cpufreq
+      job_timeout: 10
+      env: 'KSELFTEST_MAIN_SH_ARGS="-t suspend_rtc"'
+    kcidb_test_suite: kselftest.cpufreq
```

Comment:
I'm not sure if we need to define a different `kcidb_test_suite` here, as this is the same cpufreq kselftest but run with different parameters. @JenySadadia is there an existing convention for this? Users need a way to differentiate between the same test executed with different parameters once the results land in KCIDB.
(This goes back to the discussion raised [here](https://github.com/kernelci/kernelci-pipeline/issues/636))

Reply:
Sounds good to me, thanks @JenySadadia ! About the parameters passed to this test, is there any field we can add or re-use to encode this information? besides being useful for filtering results, it's also good information for manually reproducing the issues.

Reply:
One way is to store that information in node `data` field and later on assign that to `misc` field for KCIDB submission.